plugins/modules/lvg.py

```diff
@@ -227,13 +227,13 @@
 
 - name: Varyon a volume group
   ibm.power_aix.lvg:
-    vg_name: datavg
-    state: varyon
+    vg_name=datavg
```

Comment:
looks I only can add comments to the lines that modified. should we modify vg_name and state in other examples too?

Reply:
we don't have to, for this PR you only need to check the test modules. looks like i have not rebased this properly to include the latest changes in the IBM repo

Reply:
okay i have rebased properly now. you should no longer see a diff between the lvg module. all changes will be under test/unit